app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,9 @@
     android:layout_height="match_parent"
     tools:context=".ui.MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <FrameLayout
+        android:id="@+id/flContainer"
+        android:layout_width="match_parent"
```

Comment:
match_parent should not be used inside ConstraintLayout. It might be working with FrameLayout here, But this tag's behavior is really not defined in ConstraintLayout, And it is prone to (breaking) changes in the future. 

* (Use 0dp instead)